### PR TITLE
Original

### PR DESCRIPTION
**Pull request by kate-holdener**
Fixes #issue_number

**What was changed?**

The AI team produced a new version of Scantron.py, which recognizes lightly colored bubbles. That was in the AI-Fix branch. That branch had a lot of changes, and I ported only the relevant changes (parameter changes). Additionally, created a single Dockerfile that can be used to serve the front and back ends of the application with gunicorn.

**Why was it changed?**

Not all bubbles were being recognized by the application.

**How was it changed?**
Added new endpoints to AppServer.py, to serve static javascript files, that get generated by react when running 'npm build'. The dockerfile copies these static files into the static directory, which AppServer uses to serve the front end.

### CODE REVIEW

File: bubblescan-client/src/App.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ function App() {
   // Function to send message to Flask
   const sendMessage = async () => {
     try {
-      const res = await fetch("http://localhost:5001/api/message", {
+      const res = await fetch("/api/message", {
```
Comment (kate-holdener) [inline bubblescan-client/src/App.tsx:22]: @rkarmuri These are the changes that might be breaking the dev version of the app. If that's the case, you'll need to parameterize the URL, and use http://localhost:5001 for dev and just blank for prod.

Comment (rkarmuri) [reply bubblescan-client/src/App.tsx:22]: Sure!!! You are right, I will create a dev branch and parameterize the URL to accept the localhost.

**Review by rkarmuri — DISMISSED** [review]
Looks good.

**Review by rkarmuri — APPROVED** [review]
Merge conflicts resolved well.

**Pull request merged**